docs/install/ipfs-desktop.md

```diff
@@ -62,7 +62,7 @@ Or, if you'd rather use a package manager, check this [list of third-party packa
 
    ![The IPFS Desktop status bar menu in the Windows status bar.](./images/ipfs-desktop/install-windows-ipfs-desktop-status-bar.png)
 
-The IPFS Desktop application has finished installing. Now, learn how to [host a website using IPFS →](../how-to/websites-on-ipfs/single-page-website.md).
+The IPFS Desktop application has finished installing. Now, [add your site](../how-to/websites-on-ipfs/single-page-website.md#add-your-site).
```

Comment:
Looks like there should still be a `/` between `/single-page-website.md` and `#add-your-site`?

Same thing for the other link that was updated.